src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+$days-in-month: 31;
+$cell-width: 100px;
+$cell-height: 100px;
+$calendar-field: $cell-width * 7 +7;
+
 body {
+  font-family: "Arial", sans-serif;
   margin: 0;
 }
+
+.calendar {
+  box-sizing: content-box;
```

Comment:
why do you set `border-box` for all elements `*` and set `content-box` here?

Reply:
also, you should follow the checklist, create separate files for reset styles, variables as well
<img width="717" alt="image" src="https://user-images.githubusercontent.com/102411130/220044041-8f7173e8-1da4-4487-be51-f56265345530.png">